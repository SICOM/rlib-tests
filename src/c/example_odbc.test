COMMAND=./example_odbc
FORMATS="pdf xml txt csv html"
EXPECTED_pdf=
EXPECTED_xml=
EXPECTED_txt=
EXPECTED_csv=
EXPECTED_html=
