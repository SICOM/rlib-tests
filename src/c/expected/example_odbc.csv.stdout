"Number         ","Name                ","Type      ","Category  "
"10000000006    ","NRS                 ","9999      ","0         "
"10000000008    ","food sales          ","9999      ","0         "
"10000000005    ","overring            ","9999      ","0         "
"10000000007    ","refund              ","9999      ","0         "
"10000000010    ","gift cert sales     ","9999      ","0         "
"10000000011    ","other sales         ","9999      ","0         "
"10000000001    ","EI Sales            ","9999      ","0         "
"10000000002    ","TO Sales            ","9999      ","0         "
"10000000003    ","DT Sales            ","9999      ","0         "
"10000000009    ","nonfood sales       ","9999      ","0         "
"10000000013    ","credits issued      ","9999      ","0         "
"10000000014    ","promo sales         ","9999      ","0         "
"10000000015    ","gift cert redeemed  ","9999      ","0         "
"10000000060    ","reserved            ","9999      ","0         "
